applications/ContactStructuralMechanicsApplication/custom_utilities/base_contact_search.h

```diff
@@ -133,12 +135,12 @@ class TreeContactSearch
      *          -# InterfaceMapper = InterfacePreprocess()
      *          -# InterfacePart = InterfaceMapper.GenerateInterfacePart(Complete_Model_Part)
      */
-    TreeContactSearch( 
+    BaseContactSearch(
         ModelPart& rMainModelPart, 
         Parameters ThisParameters =  Parameters(R"({})") 
         ); 
     
-    virtual ~TreeContactSearch()= default;;
+    virtual ~BaseContactSearch()= default;;
```

Comment:
```suggestion
    virtual ~BaseContactSearch()= default;
```

Reply:
I will upload the changes in a future PR. I found some minor errors on OMP these days